Automattic/jetpack
# anchor-fm: set initial settings for podcast player block

Set initial settings for the podcast player block in the anchor.fm -> flow, disabling the options the following options:

* Show Art Cover
* Show Episode Title
* Show Episode Description

Fixes #18279

#### Changes proposed in this Pull Request:

* Anchor.fm: 

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Create a new podcast episode from anchor.fm to wp.com. You can re-use the data provided in the next testing URL:
`
http://localhost/wp-admin/post-new.php?action=edit&anchor_podcast=22b6608&anchor_episode=e324a06c-3148-43a4-85d8-34c0d8222138&spotify_show_url=https%3A%2F%2Fopen.spotify.com%2Fshow%2F6HTZdaDHjqXKDE4acYffoD%3Fsi%3DEVfDYETjQCu7pasVG5D73Q
`
* Confirm with the changes introduced by this PR the episode options are disabled as default:

Before | After
------|--------
![image](https://user-images.githubusercontent.com/77539/105025205-cd562300-5a2b-11eb-9068-a357c939c78d.png) | ![image](https://user-images.githubusercontent.com/77539/105024360-c549b380-5a2a-11eb-9d41-a293a5158d14.png)

Before | After
------|--------
![image](https://user-images.githubusercontent.com/77539/105025312-efe83c00-5a2b-11eb-8ead-0511b9b30150.png) | ![image](https://user-images.githubusercontent.com/77539/105025447-173f0900-5a2c-11eb-84f0-9d8531d07fc1.png)

#### Proposed changelog entry for your changes:


n/a
